src/core/biblio.js

```diff
@@ -92,39 +95,32 @@ export async function resolveRef(key) {
   return entry;
 }
 
+/** @param {Conf} conf */
 export async function run(conf) {
   const finish = () => {
     doneResolver(conf.biblio);
   };
   if (!conf.localBiblio) {
     conf.localBiblio = {};
   }
-  if (conf.biblio) {
-    let msg = "Overriding `.biblio` in config. Please use ";
-    msg += "`.localBiblio` for custom biblio entries.";
-    pub("warn", msg);
-  }
   conf.biblio = biblio;
   const localAliases = Object.keys(conf.localBiblio)
     .filter(key => conf.localBiblio[key].hasOwnProperty("aliasOf"))
     .map(key => conf.localBiblio[key].aliasOf)
     .filter(key => !conf.localBiblio.hasOwnProperty(key));
   normalizeReferences(conf);
   const allRefs = getRefKeys(conf);
-  const neededRefs = allRefs.normativeReferences
-    .concat(allRefs.informativeReferences)
-    // Filter, as to not go to network for local refs
-    .filter(key => !conf.localBiblio.hasOwnProperty(key))
-    // but include local aliases which refer to external specs
-    .concat(localAliases)
-    // remove duplicates
-    .reduce((collector, item) => {
-      if (collector.indexOf(item) === -1) {
-        collector.push(item);
-      }
-      return collector;
-    }, [])
-    .sort();
+  const neededRefs = Array.from(
```

Comment:
I hope there be something like `Array.prototype.unique()` to do this 🤯

Reply:
I wanted it too! Can we add our own with a name that won't conflict? Like `Array.prototype.rsUnique`

Reply:
That sounds too hacky 🤔